.gitignore

```
@@ -20,6 +20,7 @@ titan_api_crud
 
 # Otros
 .scannerwork
+swagger.zip
```

```suggestion
*.zip
```
Y dejar por ahí después de la L10, pofavo